index.html

```diff
@@ -7,10 +7,127 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
👏 Also, this may be helpful here: https://www.cdnplanet.com/blog/faster-google-webfonts-preconnect/